apex/transformer/parallel_state.py

```diff
@@ -17,11 +17,10 @@
 """Model and data parallel groups."""
 from typing import Tuple, Optional
 import warnings
-
+import os
 import torch
 
 from apex.transformer.log_util import get_transformer_logger
-from apex.transformer._ucc_util import HAS_UCC
```

Comment:
oh I think this shouldn't be removed.
Could you revert this deletion?

Reply:
Oops, sorry, I missed that one :sweat_smile:  Thanks!